### ISSUE TITLE
Install Issue

### ISSUE THREAD

**Issue by jcmp5858** [inkstitch/inkstitch#1473]
Hi I installed Inkstitch on my mac following the instructions. I was running into the "unverified developer issue" but after several tries the instructions online worked. However, now when I click "manage extensions" on Inkscape, this error occurs:

Traceback (most recent call last):
  File "extension-manager-bootstrap.py", line 133, in <module>
    Bootstrap().run()
  File "/Applications/Inkscape.app/Contents/Resources/share/inkscape/extensions/inkex/base.py", line 131, in run
    self.save_raw(self.effect())
  File "extension-manager-bootstrap.py", line 78, in effect
    if self.install_fallback():
  File "extension-manager-bootstrap.py", line 96, in install_fallback
    from urllib3.exceptions import NewConnectionError
ModuleNotFoundError: No module named 'urllib3'

Anyone have any ideas how to fix this?

**Comment by jcmp5858** [comment]
Ok so I'm not resolving this because I don't know if I mistakenly broke something also. However, going to the folder in "/Applications/Inkscape.app/Contents/Resources/share/inkscape/extensions/inkex" I found a folder called "elements". After playing around with it, I think my mac was not realizing the files in the "elements" folder were different than the ones in the extensions/inkex folder. So I took the files in the "elements" folder out and renamed them all starting with elements. For example one of the files was named "_filters.py" so I renamed it "elements_filters.py" After doing that, I reloaded inkscape and the Ink/stitch extension appeared!!

**Comment by kaalleen** [comment]
Did you try a manual install?

**Comment by Tina-Chris** [comment]
@jcmp5858 `I reloaded inkscape and the Ink/stitch extension appeared!!`
So you could resolve this issue?
Cheers, Tina

**Comment by kaalleen** [comment]
I guess, I wasn't very clear in what I wrote.
I was wondering about your .py files. That's why I thought you either may have the wrong download file or a manual install? But now I see that you were talking about renaming the inkex files. In the release version Ink/Stitch bundles it's inkex version by itself. This means, we do not really rely on the inkex in the inkscape extensions folder.

Also I am wondering how Ink/Stitch is interacting with the Inkscape extensions manager? We did not prepare it to be installed by it (yet?).   So this is very interesting.
